v2.1/reference/tools/download.md

```diff
@@ -7,18 +7,26 @@ category: reference
 
 本页面汇总了 TiDB 周边工具官方维护版本的下载链接。
 
-## TiDB-Binlog 和 Lightning
+## TiDB Binlog
 
-如需下载最新版本的 [TiDB-Binlog](/reference/tidb-binlog-overview.md) 或 [Lightning](/reference/tools/tidb-lightning/overview.md)，直接下载 TiDB 安装包即可，因为 TiDB-Binlog 和 Lightning 均包含在 TiDB 安装包中。
+如需下载最新版本的 [TiDB Binlog](/reference/tidb-binlog-overview.md)，直接下载 TiDB 安装包即可，因为 TiDB Binlog 包含在 TiDB 安装包中。
 
-以下表格中也提供了 Kafka 版本的 TiDB-Binlog 下载链接。
+以下表格中也提供了 Kafka 版本的 TiDB Binlog 下载链接。
 
 | 安装包 | 操作系统 | 架构 | SHA256 校验和 |
 |:---|:---|:---|:---|
-| [tidb-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz) (TiDB-Binlog, Lightning) | Linux | amd64 |[tidb-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-latest-linux-amd64.sha256)|
-| [tidb-binlog-kafka-linux-amd64.tar.gz](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.tar.gz)（Kafka 版本的 TiDB-Binlog）| Linux | amd64 |[tidb-binlog-kafka-linux-amd64.sha256](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.sha256)|
+| [tidb-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-latest-linux-amd64.tar.gz) (TiDB Binlog) | Linux | amd64 |[tidb-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-latest-linux-amd64.sha256)|
+| [tidb-binlog-kafka-linux-amd64.tar.gz](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.tar.gz)（Kafka 版本的 TiDB Binlog）| Linux | amd64 |[tidb-binlog-kafka-linux-amd64.sha256](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.sha256)|
 
-## TiDB-DM (Data Migration)
+## TiDB Lightning
+
+如需下载最新版本的 [TiDB Lightning](/reference/tools/tidb-lightning/overview.md)，点击下表中的下载链接即可。
+
+| 安装包 | 操作系统 | 架构 | SHA256 校验和 |
+|:---|:---|:---|:---|
+| [tidb-toolkit-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-toolkit-latest-linux-amd64.tar.gz) | Linux | amd64 | [tidb-toolkit-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-toolkit-latest-linux-amd64.sha256) |
```

Comment:
(Same comment as https://github.com/pingcap/docs/pull/1337/#pullrequestreview-261723369)

Reply:
In v2.1 doc, i changed the tag from latest to v2.1.0 and do not separate the TiDB Lightning from the TiDB pkg. commit: https://github.com/pingcap/docs-cn/pull/1628/commits/0b96b756224ae5a5a714ae992cdfa41d9e7b96a2
In v3.0 doc, i changed the tag from latest to v3.0.0. commit: https://github.com/pingcap/docs-cn/pull/1628/commits/48534185fb86329a0150bb372ea79691b3ee72a6